x-pack/test/api_integration/apis/management/rollup/lib/es_index.js

```diff
@@ -22,17 +22,27 @@ export const initElasticsearchIndicesHelpers = (es) => {
   };
 
   const deleteIndex = (index) => {
-    indicesCreated = indicesCreated.filter(i => i !== index);
-    return es.indices.delete({ index });
+    const indices = Array.isArray(index) ? index : [index];
+    indices.forEach((_index) => {
+      indicesCreated = indicesCreated.filter(i => i !== _index);
+    });
+    return es.indices.delete({ index: indices })
```

Comment:
I suggest passing `ignoreUnavailable: true` or `ignore: [404]` instead of doing the error filtering yourself.

Reply:
Cool, thanks for the tip!